src/test/scala/cromwell/engine/backend/BackendCallSpec.scala

```diff
@@ -19,9 +22,21 @@ class BackendCallSpec extends CromwellTestkitSpec("BackendCallSpec") {
 
   "BackendCall hash function" should {
     "not change very often" in {
-      val actual = backendCall.hash
+      val actual = backendCall.hash.futureValue.overallHash
       val expected = "8b6116bf5dad7be48066bb979b65b932"
-      assert(actual == expected,  s"Expected BackendCall hash to be $expected, but got $actual.  Did the hashing algorithm change?")
+      assert(actual == expected, s"Expected BackendCall hash to be $expected, but got $actual.  Did the hashing algorithm change?")
+    }
+
+    "not change with a Docker image hash specified" in {
+      val nameAndDigest = "ubuntu@sha256:a2c950138e95bf603d919d0f74bec16a81d5cc1e3c3d574e8d5ed59795824f47"
+      val sources = SampleWdl.CallCachingHashingWdl.asWorkflowSources( s"""runtime { docker: "$nameAndDigest" } """)
+      val descriptor = WorkflowDescriptor(WorkflowId(UUID.randomUUID()), sources)
+      val call = descriptor.namespace.workflow.calls.find(_.unqualifiedName == "t").get
+      val backendCall = backend.bindCall(descriptor, CallKey(call, None), descriptor.actualInputs, AbortRegistrationFunction(_ => ()))
+
+      val actual = backendCall.hash.futureValue.overallHash
```

Comment:
So this `Future` doesn't need to contact DockerHub/GCR because the `nameAndDigest` already includes the full hash?